data/src/test/java/com/example/data/mappers/TimezoneEntityMapperTest.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.data.mappers
+
+import com.example.data.db.tables.TimezoneEntity
+import com.example.data.modelsApi.currentWeather.CurrentWeatherApiModel
+import com.example.data.modelsApi.currentWeather.Data
+import com.example.data.modelsApi.currentWeather.Weather
+import com.example.data.utils.CityConverter
+import com.example.domain.models.CityModel
+import org.mockito.Mockito.*
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+internal class TimezoneEntityMapperTest {
+
+    @Test
+    fun returnTimezoneEntity() {
+        val cityId = 5128581
+        val cityModel = CityModel(cityId,"New York City","United States")
+
+        val cityConverter = mock(CityConverter::class.java)
+        `when`(cityConverter.getCityModelByCityId(cityId)).thenReturn(cityModel)
+
+        val timezoneEntityMapper = TimezoneEntityMapper(cityConverter)
+        val weather = Weather("800", "Clear sky", "iconURL")
+        val data = Data("New York City", 52.3, weather, "2020-09-22", "America/New_York")
+        val dataList = listOf(data)
+        val apiModel = CurrentWeatherApiModel(dataList)
+
+        assertEquals( TimezoneEntity(cityModel, "America/New_York"),
```

Comment:
It is much more readable to match 2 variables (actual and expected) instead of creating objects and calling methods in method assertEquals.
It is relevant for others tests also.

By the way, read something about rule "given when then" in tests. This rule helps to make tests more readable.

Reply:
Fixed